src/db/migrations/20190728220710-create-user.js

```diff
@@ -0,0 +1,63 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
```

Comment:
Change to es6 i.e. 
```suggestion
export default {
```
or doesn't this work? Please do the same thing for all migration and seeder files

Reply:
The module.exports was generated from sequelize by default, will have to return back the sequelize.js to the db folder for the export default to work on the migration files.